docs/add-new-route-testcase.md

```diff
@@ -0,0 +1,31 @@
+# Add New Route testcase
+
+ ## Prerequisites
+
+User has acount in the system already created by system admin.
+User is log in. 
+User is on a main view.
+
+ ## Steps
+
+ ### Step 1
+User clicks "Add New Route" button at the top of the page.
+If User use screen with lower resolution "Add New Route" button is hidden in hamburger menu.
+
+ ### Expected results 1
+User sees Add New Route view with 7 fields:
+1. "Date" - date field
+1. "Choose a car" - dropdown field
+1. "Description" - text field
+1. "From" - text field
+1. "Destination" - text field
+1. "Starting Mileage" - number field
+1. "Ending Mileage" - number field
+and 1 button "Submit" at the bottom of form.
+
+ ### Step 2
+User enters correct data
```

Comment:
You can think of validation scenario (validation for every field, with max./min. values, etc.).

Reply:
Also: "correct data" is too loose for me - you can add examples in commas for example, which might be helpful for user that would read that. These are self-explanatory, although you've already raised an issue, that you can add special characters to the milage fields (#114). That would look:

1. Date - date field, eg. 11/12/2018 (mask does matter here - order of writing the date). ISO 8601 states how the date should look like, FYI. 
1. Choose a car - dropdown field (information, that this is single-select only, not multiple values could be selected)
1. Description - text field (eg. "Lorem ipsum et dolores")